src/delegate/printer.rs

```diff
@@ -181,12 +186,16 @@ where
     N: Notification,
     N::Params: Serialize,
 {
+    #[derive(Serialize)]
+    struct RawNotification<T: Serialize> {
+        jsonrpc: Version,
+        method: String,
+        params: T,
+    }
     // Since these types come from the `lsp-types` crate and validity is enforced via the
     // `Notification` trait, the `unwrap()` calls below should never fail.
-    let output = serde_json::to_string(&params).unwrap();
-    let params = serde_json::from_str(&output).unwrap();
-    serde_json::to_string(&request::Notification {
-        jsonrpc: Some(Version::V2),
+    serde_json::to_string(&RawNotification {
+        jsonrpc: Version::V2,
         method: N::METHOD.to_owned(),
```

Comment:
By changing the `method` field to `&'static str`, we can eliminate this `to_owned()` call.